argocd-notifications/README.md

```diff
@@ -0,0 +1,81 @@
+```
+
+  As this is in initial deployment stages for this application manifests are modified manually rather than patching them from this dir. For an example, see [cluster resources](https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/cluster-management/cluster-resources.yaml).
+
+## Triggers
+
+  Triggers require two key values, a `when` condition, which states what data from argocd will start the trigger, and a `send` value which contains an array of template names to call when that trigger fires. Other fields can be added like `description`, or `oncePer`, which tells argocd to only fire the trigger one time per value for the parameter passed in. This is utilized currently in the [slack configmap](./common/argocd-notifications-cm-slack.yaml#L32) to only allow one success of sync-start per run of a Commit SHA. Because the run trigger will hit before any of the others, it fires the sync run template, and the stacks all the other sync-status notifications (such as failed, degraded, etc.) as replies. Triggers are implemented in such a way that they are notification-service agnostic, because they only listen for conditions in argocd and then call templates when those conditions resolve true.
+
+  More documentation to come, for now see [the docs](https://argocd-notifications.readthedocs.io/en/stable/triggers/).
+
+## Templates
+
+  Templates are what will generate the notification content itself, and works using the html/template golang package. Templates can be referenced by multiple triggers. Additionally one template can work for multiple services, provided everything else is also set up properly. It does this by using its `<template_name>.message` property, which will work with almost all mediums of communication (not sure about webhooks). All the other top level properties in a template are notification-service specific and will only render in that given service type (github, slack, googlechat, etc.). All of Operate-First's trigger templates and trigger names can be found [here](./common/argocd-notifications-cm-slack.yaml), and are named the same ones as the argocd-notifications catalog. They are not protected values, triggers and templates can be named anything. Additionally other values can be passed into the templates through the `data.context` value of the [configMap](./common/argocd-notifications-cm-slack.yaml), which does not come from argocd. This is useful for specifying different environments, where one can patch the overlay with argocd-server route.
+
+  More documentation and examples to come, for now see [the docs](https://argocd-notifications.readthedocs.io/en/stable/templates/).
+
+## Notification services / integrations
+
+  [The Docs](https://argocd-notifications.readthedocs.io/en/stable/services/overview/) are the best resource on this topic. Refer to them.
+
+## Deployment
+
+  Currently argocd-notifications is deployed in the argocd namespace, alongside argocd itself.
```

Comment:
Don't think we need a section for this, you can mention this in the intro with something like "we deploy argocd-notifications along side argocd etc. etc."